pkg/cmd/server/origin/run_components.go

```diff
@@ -345,10 +345,7 @@ func (c *MasterConfig) RunSDNController() {
 // RunImageImportController starts the image import trigger controller process.
 func (c *MasterConfig) RunImageImportController() {
 	osclient := c.ImageImportControllerClient()
-	factory := imagecontroller.ImportControllerFactory{
-		Client: osclient,
-	}
-	controller := factory.Create()
+	controller := imagecontroller.NewImportController(osclient, osclient, 10, 2*time.Minute)
```

Comment:
I'll review it later today when I'm back online.
On Dec 18, 2015 5:54 PM, "Jordan Liggitt" <notifications@github.com> wrote:

> In pkg/cmd/server/origin/run_components.go
> <https://github.com/openshift/origin/pull/6407#discussion_r48045237>:
>
> > @@ -345,10 +345,7 @@ func (c *MasterConfig) RunSDNController() {
> >  // RunImageImportController starts the image import trigger controller process.
> >  func (c *MasterConfig) RunImageImportController() {
> >  	osclient := c.ImageImportControllerClient()
> > -	factory := imagecontroller.ImportControllerFactory{
> > -		Client: osclient,
> > -	}
> > -	controller := factory.Create()
> > +	controller := imagecontroller.NewImportController(osclient, osclient, 10, 2*time.Minute)
>
> cast osclient to the interfaces the function is expecting for sanity
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6407/files#r48045237>.
>